org/enigma/EnigmaRunner.java

```diff
@@ -397,27 +395,27 @@ public EnigmaSettings getInstance()
 
 	public void populateMenu()
 		{
-		//stopb = new JButton(); //$NON-NLS-1$
-		//stopb.addActionListener(this);
-		//stopb.setToolTipText(Messages.getString("EnigmaRunner.MENU_STOP"));
-		//stopb.setIcon(LGM.getIconForKey("EnigmaPlugin.STOP"));
-		//LGM.tool.add(stopb, 5);
+		stopb = new JButton(); //$NON-NLS-1$
+		stopb.addActionListener(this);
+		stopb.setToolTipText(Messages.getString("EnigmaRunner.MENU_STOP"));
+		stopb.setIcon(LGM.getIconForKey("EnigmaPlugin.STOP"));
+		LGM.tool.add(stopb, 5);
 		runb = new JButton(); //$NON-NLS-1$
 		runb.addActionListener(this);
 		runb.setToolTipText(Messages.getString("EnigmaRunner.MENU_RUN"));
 		runb.setIcon(LGM.getIconForKey("EnigmaPlugin.EXECUTE"));
-		LGM.tool.add(runb, 5);
+		LGM.tool.add(runb, 6);
```

Comment:
Can we use `++num` like scientists?

Bigger-picture question for another time: why are we passing this at all?

Reply:
Alright 1e8ee27d77606d09ac177822075d5c635de00dd1 it's done. Also it's here because the LGM toolbar already exists at this point and lgmplugin is just wanting to "insert" its buttons into place. LGM had a lack of standardization around how plugins were really supposed to modify the UI.